extensions/wikia/Listusers/SpecialListusers_helper.php

```diff
@@ -89,41 +86,36 @@ function setOrder    	( $orders = array() ) {
 				}
 			}
 		}
+
+		// SUS-3207 - order by number of edits (descending) by default
 		if ( empty( $this->mOrder ) ) {
-			$this->mOrder[] = 'user_name ASC';
+			$this->mOrder[] = 'edits DESC';
 		}
 	}
 
-	function getFilterGroup () { return $this->mFilterGroup; }
 	function getGroups   	() { return $this->mGroups; }
-	function getUserName	() { return $this->mUserName; }
-	function getEdits    	() { return $this->mEdits; }
-	function getLimit    	() { return $this->mLimit; }
-	function getOffset   	() { return $this->mOffset; }
-	function getOrder    	() { return $this->mOrder; }
 
 	public function loadData() {
-		global $wgMemc, $wgLang, $wgUser, $wgDBname;
+		global $wgMemc, $wgLang, $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app